Chore:(Docs) Add TS version of Svelte snippet(get-started)

miily8310s opened the pull request:
Closes #

<!-- Thank you for contributing to Storybook! If your PR is related to an issue, provide the number(s) above; if it resolves multiple issues, be sure to break them up (e.g. "closes #1000, closes #1001"). -->

## What I did

<!-- Briefly describe what your PR does -->
Add TS version of Svelte snippet(only on get-started page)

## How to test

<!-- Please include the steps to test your changes here. For example:

1. Run a sandbox for template, e.g. `yarn task --task sandbox --start-from auto --template react-vite/default-ts`
2. Open Storybook in your browser
3. Access X story

-->

## Checklist

<!-- Please check (put an "x" inside the "[ ]") the applicable items below to make sure your PR is ready to be reviewed. -->

- [x] Make sure your changes are tested (stories and/or unit, integration, or end-to-end tests)
- [x] Make sure to add/update documentation regarding your changes
- [ ] If you are deprecating/removing a feature, make sure to update
      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

- [ ] If this PR should be tested against many or all sandboxes,
      make sure to add the `ci:merged` or `ci:daily` GH label to it.
- [ ] Make sure this PR contains **one** of the labels below.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`

<!--

Everybody: Please submit all PRs to the `next` branch unless they are specific to the current release. Storybook maintainers cherry-pick bug and documentation fixes into the `main` branch as part of the release process, so you shouldn't need to worry about this. For additional guidance: https://storybook.js.org/docs/react/contribute/how-to-contribute

-->

miily8310s committed:
Add TS version of svelte snippet(get-started)

miily8310s committed:
fix: semi-colon

jonniebigodes commented on docs/snippets/svelte/button-story-with-args.ts-4-9.mdx:
```diff
@@ -0,0 +1,35 @@
+ * to learn how to use render functions.
+ */
+export const Primary: Story = {
+  render: (args) => ({
+    Component: Button,
+    props: args,
+  }),
```
@miily8310s, we can drop the render function and comment in this example and the pre-TS variation with the same name.

jonniebigodes commented on docs/snippets/svelte/histogram-story.ts-4-9.mdx:
```diff
@@ -0,0 +1,37 @@
+ * to learn how to use render functions.
+ */
+export const Default: Story = {
+  render: (args) => ({
+    Component: Histogram,
+    props: args,
+  }),
```
@miily8310s the same applies here. We can drop the render function and comment for this and the pre-TS-4.9 variation

jonniebigodes commented on docs/snippets/svelte/your-component.ts-4-9.mdx:
```diff
@@ -0,0 +1,35 @@
+ * to learn how to use render functions.
+ */
+export const FirstStory: Story = {
+  render: (args) => ({
+    Component: YourComponent,
+    props: args,
+  }),
```
@miily8310s same rule applies here and for the follow-up pre-TS-4.9 variation

jonniebigodes reviewed (changes requested):
@miily8310s, thanks for taking the time to put together this pull request and complement our existing examples. Appreciate it greatly 🙏 ! I left a couple of items for you to look at when you have a chance. For context, I'm aware that the existing Javascript variations for Svelte mirroring this pull request include explicit `render` functions, but for these examples, we don't need them, and if you're ok we can adjust them on a follow-up pull request. Would you be interested in it?

Let me know, and we'll go from there.

Looking forward to hearing from you and getting this one in.

Have a great day.

Stay safe

miily8310s commented:
> if you're ok we can adjust them on a follow-up pull request. Would you be interested in it?

OK

jonniebigodes commented:
@miily8310s, that's great to hear. Glad that you're willing to do it 🙏 ! 

Let me know once you've addressed the feedback so that we can get this one in and go from here for the next one.

Once again, thanks for this great contribution ❤️

miily8310s committed:
fix: reflect feedback for render function

miily8310s commented:
@jonniebigodes 

> Let me know once you've addressed the feedback so that we can get this one in and go from here for the next one.

I'm sorry for responding slowly. I'd reflected your feedback, please check my modification!

jonniebigodes committed:
Merge branch 'next' into code-snippets-for-svelte

jonniebigodes commented:
@miily8310s, no need to be sorry whatsoever. I understand it. I checked the changes, and they're great. Once again thanks for addressing the feedback so promptly. Appreciate it 🙏 ! I'm going to go ahead and merge this one. And if you're interested in contributing to the changes for the JS versions, let me know once the pull request is up, and I'll be more than glad to follow up with you and we'll go from there.

Hope you have a great week.

Stay safe
